sentences/da/binary_sensor_HassGetState.yaml

```
@@ -0,0 +1,911 @@
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: motion
+          state: "on"
+
+      - sentences:
+          - "er alle bevægelse[s][ ]sensorer {bs_motion_states:state} [i <område>]"
+        response: all
+        slots:
+          domain: binary_sensor
+          device_class: motion
+
+      - sentences:
+          - "hvilke[n] bevægelse[s][ ]sensor[er] er {bs_motion_states:state} [i <område>]"
+        response: which
+        slots:
+          domain: binary_sensor
+          device_class: motion
+
+      - sentences:
+          - "<hvormange> bevægelse[s][ ]sensor[er] er {bs_motion_states:state} [i <område>]"
+        response: how_many
+        slots:
+          domain: binary_sensor
+          device_class: motion
+
+      # Occupancy
+      - sentences:
+          - "er <navn> {bs_occupancy_states:state} [i <område>]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: occupancy
+        slots:
+          domain: binary_sensor
+          device_class: occupancy
+
+      - sentences:
+          - "er [der] [<nogen>] optaget[ ]sensorer {bs_occupancy_states:state} [i <område>]"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: occupancy
+
+      - sentences:
+          - "er optaget detekteret [i <område>]"
+          - "er optaget registreret [i <område>]"
+          - "er noget optaget [i <område>]"
+          - "er <område> optaget"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: occupancy
+          state: "on"
+
+      - sentences:
+          - "er alle optaget[ ]sensorer {bs_occupancy_states:state} [i <område>]"
+        response: all
+        slots:
+          domain: binary_sensor
+          device_class: occupancy
+
+      - sentences:
+          - "hvilke[n] optaget[ ]sensor[er] er {bs_occupancy_states:state} [i <område>]"
+        response: which
+        slots:
+          domain: binary_sensor
+          device_class: occupancy
+
+      - sentences:
+          - "<hvormange> optaget[ ]sensor[er] er {bs_occupancy_states:state} [i <område>]"
+        response: how_many
+        slots:
+          domain: binary_sensor
+          device_class: occupancy
+
+      # Opening
+      - sentences:
+          - "(er|står) <navn> {bs_opening_states:state} [i <område>]"
+        response: one_yesno
+        requires_context:
+          domain: binary_sensor
+          device_class: opening
+        slots:
+          domain: binary_sensor
+          device_class: opening
+
+      - sentences:
+          - "(står|er) [der] <nogen> åbning[er] {bs_opening_states:state} [i <område>]"
+        response: any
+        slots:
+          domain: binary_sensor
+          device_class: opening
+
+      - sentences:
+          - "(er|står) alle åbning[er] {bs_opening_states:state} [i <område>]"
```

Same as above, `står` doesn't feel right here.

```suggestion
          - "er alle åbning[er] {bs_opening_states:state} [i <område>]"
```